x/concentrated-liquidity/lp.go

```diff
@@ -77,6 +77,13 @@ func (k Keeper) createPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddr
 		}
 	}
 
+	if !k.hasFullPosition(ctx, poolId, owner, lowerTick, upperTick, joinTime, freezeDuration) {
```

Comment:
@p0mvn not sure why we have to check if the position is frozen or not as you suggested?

Reply:
talked to @AlpinYukseloglu  about this and here's a quick tldr is 

"currently we donot support 0sec uptime therefore positions are not frozen from 0sec-60sec (min supported uptime is 1min). I think in future we are planning on adding 0sec uptime so that all positions are frozen to begin with" 

for the sake of this PR I added frozen check and made a note to remove it once we add 0sec supported uptime